### ISSUE TITLE
Update plugin with new RFC implemented.

### ISSUE THREAD

**Issue by wilfredwee** [MisterTea/hyper-htm#1]
From the looks of it, the necessary API for HTM support in Hyper is implemented: https://github.com/zeit/hyper/pull/3464

Can the plugin be updated and productionized (with installation instructions etc.) now? It'd be super cool to finally have an alternative to iTerm2 + tmux -CC :)

**Comment by MisterTea** [comment]
Oh interesting, I'll take a look later on this week

**Comment by MisterTea** [comment]
Hey, it looks like that API doesn't let me choose where to create/delete windows.  It just creates one based on which session is active.  Sorry, but this is still not feasible with the current hyper codebase.

**Comment by MisterTea** [comment]
Actually I'll make another PR and we'll see what happens

**Comment by MisterTea** [comment]
@wilfredwee Alright it's updated.  Once we get the hyper hook in I can work on productionizing.

**Comment by wilfredwee** [comment]
Great! Let's hope the Hyper devs are responsive.

**Comment by msemikin** [comment]
Is there still anything blocking proper integration of the plugin? Would love to test it out together with ET

**Comment by MisterTea** [comment]
@msemikin I'm just waiting for the next Hyper release, but you can use it today if you build hyper from source. Just add the plugin and then run "htm" locally to test it out

**Comment by nrgapple** [comment]
Any update on this?

**Comment by brian-avery** [comment]
This looks awesome @MisterTea. Any idea when this will land? Thanks!

**Comment by MisterTea** [comment]
I'm still waiting for a new version of Hyper.js to come out.  Once the new version is out, this plugin should work and I can start promoting it and polishing the code.

**Comment by MisterTea** [comment]
This plugin requires Hyper 3.1.x

**Comment by brian-avery** [comment]
Awesome. Thanks. :partying_face:  

I just saw that this issue hadn't been updated since 2019. I didn't realize that you were still waiting on the release.

**Comment by FranklinYu** [comment]
Hyper 3.1 is released! 🎉

**Comment by MisterTea** [comment]
This plugin is up to date now :-D
